src/main/java/org/osc/manager/ism/entities/VirtualSystemElementImpl.java

```diff
@@ -0,0 +1,81 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.manager.ism.entities;
+
+import org.osc.sdk.manager.element.ApplianceSoftwareVersionElement;
+import org.osc.sdk.manager.element.DistributedApplianceElement;
+import org.osc.sdk.manager.element.DomainElement;
+import org.osc.sdk.manager.element.VirtualSystemElement;
+import org.osc.sdk.manager.element.VirtualizationConnectorElement;
+
+public class VirtualSystemElementImpl implements VirtualSystemElement {
```

Comment:
move the impls to a different package. These impls are not entities and as such should not be in the entities package.

Reply:
I will create model package and add this interface impl. In this package.